packages/drivers/odsp-driver/src/odspDocumentDeltaConnection.ts

```diff
@@ -257,6 +248,21 @@ export class OdspDocumentDeltaConnection extends DocumentDeltaConnection impleme
 
     private socketReference: SocketReference | undefined;
 
+    /**
+     * Error raising for socket.io issues
+     */
+    protected createErrorObject(handler: string, error?: any, canRetry = true): DriverError {
```

Comment:
Where is this getting called? I see that previously we were converting to "errorObjectFromSocketError"" and that now we have removed it and added it in here. But I cann't see where is this called?

Reply:
All the usages are in this PR. Search for this.createErrorObject calls